regulations/templatetags/paragraphs.py

```diff
@@ -6,7 +6,14 @@
 
 @register.filter(name='pdepth')
 def pdepth(value):
-    depth = len(value.get("label", []) or []) - section_depth
-    if len(value.get("marker", []) or []) > 1:
-        depth = depth - (len(value.get("marker", []) or []) - 1)
-    return depth
+    parent_type = value.get("parent_type", "")
+    label_len = len(value.get("label", []))
+    marker_len = len(value.get("marker", []) or [])
+    depth = label_len - section_depth
+
+    if parent_type == "appendix" or label_len == 0:
```

Comment:
Should be able to remove the reference to `appendix`. may be able to go even further and simply say at the end that `if depth < 1, return 1, otherwise return depth` (basically return a depth of at least one.